doc/source/whatsnew/v1.2.0.rst

```diff
@@ -257,7 +255,7 @@ Conversion
 
 Strings
 ^^^^^^^
-
+- Bug in :meth:`Series.to_string` adding a leading space when ``index=False`` (:issue:`24980`)
```

Comment:
can you also that this would affect `DataFrame.to_latex` and `DataFrame.to_string`

Reply:
> can you also that this would affect `DataFrame.to_latex` and `DataFrame.to_string`

"also" waht? Do you mean make sure this PR dosen't affect `DataFrame.to_latex` and `DataFrame.to_string`?

Reply:
```suggestion
- Bug in :meth:`Series.to_string`, :meth:`DataFrame.to_string`, and :meth:`DataFrame.to_latex` adding a leading space when ``index=False`` (:issue:`24980`)
```

Reply:
updated :)